### PR TITLE
Implement Tier formula

### PR DESCRIPTION
**Pull request by BlueWeabo**
Implements the weighted tier formula discussed in the dev channel. Weights of each parts are only temporary and can be changed.

### CODE REVIEW

File: src/main/java/gregtech/api/multitileentity/multiblock/base/PowerController.java [hunk 1]
```diff
@@ -68,9 +67,10 @@ protected boolean hasPerfectOverclock() {
     }
 
     protected void updatePowerLogic() {
-        power.setEnergyCapacity(GT_Values.V[tier] * amperage * 2 * MINUTE);
+        long voltage = (long) Math.ceil(Math.pow(2, 4 + tier));
```
Comment (chochem) [inline src/main/java/gregtech/api/multitileentity/multiblock/base/PowerController.java:70]: its totally possible I am misunderstanding what this does, but shouldnt it be
(long) Math.ceil(Math.pow(2, 3 + 2*tier))
?
aka 8x4^tier or 2^(3+2xtier)

Comment (chochem) [reply src/main/java/gregtech/api/multitileentity/multiblock/base/PowerController.java:70]: basically 8x4 =32, 8x4^2=128, 8x4^3=512 but now allowing floats

Comment (BlueWeabo) [reply src/main/java/gregtech/api/multitileentity/multiblock/base/PowerController.java:70]: I knew doing this late in the night wasn't the best idea, but if this is the only thing I got wrong it wasn't bad. but yeah. I forgot to do the formula in my head for a bit to see if it works

File: src/main/java/gregtech/api/multitileentity/machine/MultiTileBasicMachine.java [hunk 2]
```diff
@@ -762,9 +762,9 @@ public ForgeDirection getPowerOutputSide() {
     }
 
     protected void updatePowerLogic() {
-        power.setEnergyCapacity(GT_Values.V[tier] * power.getMaxAmperage() * 2 * MINUTE);
-        power.setMaxVoltage(GT_Values.V[tier]);
-        power.setMaxAmperage(1);
+        long voltage = (long) Math.ceil(Math.pow(2, 3 + (2 * tier)));
```
Comment (Glease) [inline src/main/java/gregtech/api/multitileentity/machine/MultiTileBasicMachine.java:765]: wouldn't this lead up to max voltage like 8193 when tier is not an integer? can other part of the system handle such a thing? or is this how it is supposed to be done?

**Comment by BlueWeabo** [comment]
Uhm. forgot about this. closing. nope not fixing it rn

**Pull request closed without merging**